docs/extend/manifest.md

```diff
@@ -0,0 +1,239 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/extend/"
+]
+title = "Plugin manifest"
+description = "How develop and use a plugin with the managed plugin system"
+keywords = ["API, Usage, plugins, documentation, developer"]
+advisory = "experimental"
+[menu.main]
+parent = "engine_extend"
+weight=1
++++
+<![end-metadata]-->
+
+# Plugin manifest version 0
```

Comment:
This is the title of the page. Not a regular sentence. https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md for example capital cases all words in the title.